### DISCUSSION TITLE
Connect to DB via access token and not username and password

### DISCUSSION THREAD

**Discussion by oriarazi** [prisma/prisma#14684] [Q&A]
Hey there,
I am using azure cloud services, and using that I have my postgresql db there.
I'd like to connect the Database via AAD managed identity, and not the regular way with username and password, for many reasons,
but I could not find any option doing it with Prisma framework.
Basically, the proccess will now be, app getting access token, and sending it in password field, so I cannot set it in the ENV file, as it is dynamic.
Is it Possibe? If not, I think it is a crucial feature.
Thanks

**Comment by austincrim** [comment]
Hey @oriarazi 👋,

We have an [outstanding feature request ](https://github.com/prisma/prisma/issues/13853)for this, and it looks like some of the work has been started already.

Feel free to add a 👍 or your specific use case in that issue, if you feel it hasn't been covered already.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
